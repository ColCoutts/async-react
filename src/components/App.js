import React from 'react';
import Characters from './characters/Characters';

export default function App() {
  const characters = [
    {
      name: 'Colin',
      gender: 'Alien',
      age: 'Some age',
      films: ['A movie', 'Another Movie']
    },
    {
      name: 'Chad',
      gender: 'Alien',
      age: 'Some age',
      films: ['A movie', 'Another Movie']
    },
    {
      name: 'Carl',
      gender: 'Alien',
      age: 'Some age',
      films: ['A movie', 'Another Movie']
    }
  ];

  return (
    <Characters characters={characters} />
  );
}
