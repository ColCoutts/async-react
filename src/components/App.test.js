import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App component', () => {
  it('renders snapshot', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
