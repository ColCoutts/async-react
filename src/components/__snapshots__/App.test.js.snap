// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders snapshot 1`] = `
<AllCharacters
  page={1}
/>
`;
